src/app/layout.tsx

```diff
@@ -1,17 +1,12 @@
-import type { Metadata } from "next";
-import { Poppins } from "next/font/google";
+"use client"
```

Comment:
Remove 'use client' here. The layout should be server side. See how you can achieve this.
Refer to Nextjs documentation about server side and client side components. You may also need to look into providers, in your case it may store provider or something.